js/usage-stats-chart.js

```diff
@@ -71,35 +73,24 @@
 			},
 		},
 		scales: {
-			xAxes: [
-				{
-					gridLines: {
-						color: 'rgba(0,0,0,0.05)',
-						drawTicks: false,
-					},
+			x: {
+				grid: {
+					color:
+						pkpUsageStats.config.chartType === 'bar'
+							? 'transparent'
+							: 'rgba(0,0,0,0.05)',
+					drawTicks: false,
```

Comment:
Oki.. looking pretty much as before, right? :-)

Reply:
Here's the comparison of the chart displays...
### **Bar Chart**
![image](https://github.com/user-attachments/assets/063a45fe-a438-46b6-b6a7-f6b1752bf619)


### **Line Chart**
![image](https://github.com/user-attachments/assets/5d0198c8-1d15-4651-a914-9800a157a5f5)

Reply:
Thank you for the screenshots - will also tag @Devika008 for feedback. The images with newer versions would be how it would look going forward. Please let us know if you see something worrying.

Personally having the color filled under the line chart makes it nicer. But lets see what Devika thinks.

Reply:
Hi @jardakotesovec and @blesildaramirez,

The graphs look great to me. Honestly, to me too having the colour under the line graphs makes it easier to read the line graphs. But what helps more is that having a darker colour line at the end of the coloured line graph like the image below. makes it super easy to read.

![image](https://github.com/user-attachments/assets/2eb06dfb-c83a-46b6-a0e6-df9f92d73cb8)

Reply:
Hi @Devika008 , thanks for your inputs. I updated the line chart and it now looks like this:
![image](https://github.com/user-attachments/assets/939b5541-29a9-4dc8-8bec-95231e014043)
Please let me know if this looks good from your end and I will update this PR. Thanks!